freeipa/src/main/java/com/sequenceiq/freeipa/service/freeipa/user/UserService.java

```diff
@@ -69,7 +71,7 @@
     private static final int DEFAULT_MAX_SUBJECTS_PER_REQUEST = 10;
 
     @VisibleForTesting
-    @Value("${freeipa.syncoperation.max-subjects-per-request:10}")
+    @Value("${freeipa.usersync.max-subjects-per-request:10}")
```

Comment:
It has 10 as default value and also assigned the same value as a final static int

Reply:
I removed the default from the @Value annotation. It's useful to have the `DEFAULT_MAX_SUBJECTS_PER_REQUEST` so that a value is always assigned, even if the class is not instantiated through spring (i.e., for the test)